spring-cloud-netflix-core/pom.xml

```diff
@@ -199,4 +199,25 @@
 			<optional>true</optional>
 		</dependency>
 	</dependencies>
+	<profiles>
+		<profile>
+			<id>java8plus</id>
+				<activation>
+					<jdk>[1.8,2.0)</jdk>
```

Comment:
We build with java8 by default, but have to have java 7 compatibility. I'm guessing this breaks that?